src/SSHDebugPS/PipeConnection.cs

```diff
@@ -129,7 +78,7 @@ private string GetFullPath(string path)
             string pwd;
             ExecuteCommand("pwd", Timeout.Infinite, throwOnFailure: true, commandOutput: out pwd);
             ExecuteCommand($"cd \"{path}\"; pwd", Timeout.Infinite, throwOnFailure: true, commandOutput: out fullpath);
-            ExecuteCommand($"cd \"{pwd}\"", Timeout.Infinite, throwOnFailure: false, commandOutput: out output); //This might fail in some instances, so ignore a failure and ignore output
+            ExecuteCommand($"cd '{pwd}'", Timeout.Infinite, throwOnFailure: false, commandOutput: out output); //This might fail in some instances, so ignore a failure and ignore output
```

Comment:
Why did this change and not the one above it too?

Reply:
It was during some testing. I will change it to match 

---
In reply to: [296972331](https://github.com/microsoft/MIEngine/pull/884#discussion_r296972331) [](ancestors = 296972331)